app/src/androidTest/java/blockeq/com/stellarwallet/utils/AccountUtilsTest.java

```diff
@@ -116,7 +162,7 @@ public void backwards_compatibility_test_login() {
         CipherWrapper cipherWrapper = new CipherWrapper("RSA/ECB/PKCS1Padding");
 
         assert masterKey != null;
-        String encryptedPhrase = cipherWrapper.encrypt(mnemonic + " "  + passPhrase, masterKey.getPublic(), false);
+        String encryptedPhrase = cipherWrapper.encrypt(mnemonic12 + " "  + passPhrase, masterKey.getPublic(), false);
```

Comment:
this is not he old way of encrypting mnemoni + passphrasE?

Reply:
should u write a new test for the passphrase encryption only?
`cipherWrapper.encrypt(passPhrase)`

Reply:
yeah, this entire problem only exists in the old wallet lol, cause we are using a space separating the mnemonic and passphrase :P

Reply:
never-mind, I got confused